3.Tests/UI/Hooks/Hooks.cs

```diff
@@ -1,6 +1,9 @@
 ﻿using System;
+using System.Linq;
 using System.Text.Json;
+using OpenQA.Selenium;
 using RestSharp;
+using Sprache;
```

Comment:
What is this package needed for? if it has to be installed, you should also commit the .csproj with the respective package reference

Reply:
This library i was using to name the png file, but i ended up using linq. Sprache is used for parsing. As now im not using it, its  deleted